1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,67 @@
-// Write your code here
+let images = [
+  {
+    src: "./images/Amsterdam.png",
+  },
+  {
+    src: "./images/Moscow.png",
+  },
+  {
+    src: "./images/Rome.png",
+  },
+  {
+    src: "./images/paris.png",
+  },
+  {
+    src: "./images/stockholm.png",
+  },
+  {
+    src: "./images/Madrid.png",
+  },
+  {
+    src: "./images/Berlin.png",
+  },
+];
+
+let carouselImg = document.querySelector("#carousel-img");
+
+let i = 0; 
+
+function back() {
+  if (i <= 0) {
+    i = images.length;
+  }
+  i--;
+  carouselImg.setAttribute("src", images[i].src);
+}
+
+function forward() {
+  if (i >= images.length - 1) {
+    i = -1;
+  }
+  i++;
+  carouselImg.setAttribute("src", images[i].src);
+}
+
+let backButton = document.querySelector("#back-btn");
```

Comment:
nice clear function names & variable names used for your slideshow, well done. The actually slideshow functionality works well, just a shame can't see the images at the moment!